plugins/modules/cloud/misc/proxmox_disk.py

```diff
@@ -0,0 +1,726 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2022, Castor Sky (@castorsky) <csky57@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: proxmox_disk
+short_description: Management of a disk of a Qemu(KVM) VM in a Proxmox VE cluster.
+version_added: 5.5.0
```

Comment:
5.5.0 is already out, but 5.6.0 will be released in two days:
```suggestion
version_added: 5.6.0
```